src/services/index.ts

```diff
@@ -159,30 +159,35 @@ export const fetchTokenData = async ({
       getTokenFromErc20(params),
     ])
 
-    if (!tokenInExchange)
+    if (!tokenInExchange) {
+      // Token is not registered in the contract
       return {
         token: erc20Token,
         reason: erc20Token ? TokenFromExchange.NOT_REGISTERED_ON_CONTRACT : TokenFromExchange.NOT_ERC20,
         tokenAddress: params.tokenAddress,
       }
-
-    if (!erc20Token)
+    } else if (!erc20Token) {
```

Comment:
Why add `else` if we have `return` above anyway?

Reply:
I just thought in this case it would be nice to show there's 3 cases, and they are all covered

not a big change. Consider a case where u accidentally remove the word "return" in one of the branches, it would still compile, but it would be an error